includes/model/class-crud-cpt.php

```diff
@@ -88,6 +89,27 @@ protected function is_private() {
 		return false;
 	}
 
+	/**
+	 * Wrapper function for deleting
+	 *
+	 * @param boolean $force_delete  Should the data be deleted permanently.
+	 *
+	 * @return boolean
+	 * @throws UserError  Not authorized.
```

Comment:
`@return` is the last item in the PHPDoc.

Reply:
:thinking: Noted. Documentation is sort of a mess right now. I was gonna focus on it after the release. I've actually been putting the `@throws` after the `@return` through the entire repository. It'll be better address in it's own PR, I'll keep for now for consistency.